### PR TITLE
Changes provincials

### PR DESCRIPTION
**Pull request by LuisFerArredondo**
## Describe your changes

- [ ] Bug fix
- [ ] New feature

Explain what you did:
> ...

## Issue ticket (add number)

> https://github.com/FRC2706/2024-2706-Robot-Code/issues/...

## Checklist before requesting a review
- [ ] I have performed a self-review of my code.
- [ ] I have tested thoroughly.
- [ ] I have written documentation and explained about my changes to the best of my ability.

### CODE REVIEW

File: src/main/java/frc/robot/Config.java [hunk 1]
```diff
@@ -456,9 +456,9 @@ public static enum ArmSetPoints {
   //@todo: to be calibrated
   IDLE(61),
   INTAKE(-1.1),
-  SPEAKER_KICKBOT_SHOT(12.9),
+  SPEAKER_KICKBOT_SHOT(13.9),
   NO_INTAKE(2.2),
-  SPEAKER_VISION_SHOT(30.2),
+  SPEAKER_VISION_SHOT(30.2), // 31.2
```
Comment (weizhang-2706) [inline src/main/java/frc/robot/Config.java]: My impression with vision shot at the practice field looks good. May not need to be adjusted.

File: src/main/java/frc/robot/Robot.java [hunk 2]
```diff
@@ -135,6 +135,7 @@ public void autonomousInit() {
 
     SwerveSubsystem.getInstance().setVoltageCompensation(true);
     ArmSubsystem.getInstance().resetProfiledPIDController();
+    //ShooterSubsystem.getInstance().changeCurrentLimit(true);
```
Comment (weizhang-2706) [inline src/main/java/frc/robot/Robot.java:138]: cleanup this line

File: src/main/java/frc/robot/Robot.java [hunk 3]
```diff
@@ -153,6 +154,8 @@ public void teleopInit() {
     // teleop starts running. If you want the autonomous to
     // continue until interrupted by another command, remove
     // this line or comment it out.
+    //ShooterSubsystem.getInstance().changeCurrentLimit(false);
```
Comment (weizhang-2706) [inline src/main/java/frc/robot/Robot.java:157]: clean up

File: src/main/java/frc/robot/subsystems/ShooterSubsystem.java [hunk 4]
```diff
@@ -103,7 +103,14 @@ public ShooterSubsystem() {
     public double getVelocityRPM() {
         return m_encoder.getVelocity();
     }
-
+/*
```
Comment (weizhang-2706) [inline src/main/java/frc/robot/subsystems/ShooterSubsystem.java:106]: clean up these lines

**Review by weizhang-2706 — CHANGES_REQUESTED** [review]

**Review by ErikCald — APPROVED** [review]

**Review by weizhang-2706 — APPROVED** [review]

**Pull request merged**